Gemfile.lock

```diff
@@ -229,9 +227,12 @@ DEPENDENCIES
   sinatra (= 1.4.7)
   slop (= 3.4.5)
   statsd-ruby (~> 1.3.0)
-  test-unit
+  test-unit-minitest (~> 0.9.1)
   timecop (= 0.8.0)
   unf (= 0.1.4)
   unicorn (= 5.1.0)
   webmock (~> 1.24)
   whenever (~> 0.9.4)
+
+BUNDLED WITH
```

Comment:
Ok, after reading up on this, we really think should keep it:

> Here’s the long explanation: Bundler 1.10 adds the BUNDLED WITH section to the lock, while Bundler 1.9 removes it. When a team includes some developers on 1.9 or older, and some developers on 1.10 or higher, it’s easy to end up in a situation with commits changing the lock back and forth. The simplest way to stop that problem is to upgrade to 1.10 by running gem install bundler. **When Bundler 1.10 or higher sees a BUNDLED WITH section, it will leave it in the file.**

(Emphasis mine)

http://bundler.io/blog/2015/06/24/version-1-10-released.html

Anyhow as we discussed earlier, you should never cherry-pick from `Gemfile.lock`.

What are the reasons you wouldn't want this committed @jackscotti?